docs/zh/data_source/catalog/iceberg_catalog.md

```diff
@@ -483,12 +483,20 @@ Iceberg Catalog 从 3.0 版本起支持 Google GCS。
 
 #### MetadataUpdateParams
 
-指定元数据缓存策略的一组参数。此组参数为可选。
+指定元数据缓存更新策略的一组参数。StarRocks 根据该策略更新缓存的 Iceberg 元数据。此组参数为可选。
 
-当前仅包含 `enable_iceberg_metadata_cache` 一个参数，用于指定是否缓存 Iceberg 表指针和分区名相关的数据。该参数自 3.2.1 版本起支持：
+StarRocks 默认采用[自适应元数据检索方案](#附录自适应元数据检索方案)，开箱即用。因此，一般情况下，您可以忽略 `MetadataUpdateParams`，无需对其中的策略参数进行调优。
 
-- 在 3.2.1 到 3.2.3 版本，该参数默认值统一为 `true`。
-- 自 3.2.4 版本起，如果 Iceberg 集群的元数据服务为 AWS Glue，该参数默认值仍为 `true`，如果 Iceberg 集群的元数据服务为 Hive Metastore（简称 HMS）或其他，则该参数默认值变更为 `false`。
+如果 Iceberg 数据更新频率较高，那么您可以对这些参数进行调优，从而优化自动异步更新策略的性能。
+
+| 参数                                          | 默认值                  | 说明                                                         |
+| :-------------------------------------------- | :-------------------- | :----------------------------------------------------------- |
+| enable_iceberg_metadata_cache                 | 对于 AWS Glue 为 true，对于 HMS 或其他为 false | 是否缓存 Iceberg 相关的元数据，包括 Table Cache，Partition Name Cache，Data File Cache 和 Delete Data File Cache。该参数自 3.2.1 版本起支持：<ul><li>在 3.2.1 到 3.2.3 版本，该参数默认值统一为 `true`。</li><li>自 3.2.4 版本起，如果 Iceberg 集群的元数据服务为 AWS Glue，该参数默认值仍为 `true`，如果 HMS 或其他，则该参数默认值变更为 `false`。</li></ul> |
+| iceberg_meta_cache_ttl_sec                    | 48 * 60 * 60          | 除 Table Cache 外，所有元数据缓存的 Time-To-Live（TTL）。单位：秒。 |
+| iceberg_local_planning_max_slot_bytes         | 8 * 1024 * 1024       | Iceberg 每个线程能够处理的 Manifest 文件的最大大小。单位：Byte。 |
```

Comment:
It is best to make users understand that this is to control the 'planning mode' and not that the actual thread can only handle files of this size.

Reply:
Perhaps it would be better not to expose such a complex parameter.